src/components/Review.jsx

```diff
@@ -1,21 +1,25 @@
 import React, { useEffect, useState } from "react";
-import { List, Container, Divider, Grid } from "semantic-ui-react";
+import { List, Container, Grid } from "semantic-ui-react";
 import axios from "axios";
 import "../css/Review.css";
+import Preview from "./Preview";
 
 const Review = () => {
   const [unpublishedArticleList, setUnpublishedArticleList] = useState([]);
+  const [selectedArticle, setSelectedArticle] = useState();
+
   useEffect(() => {
     const fetchUnpublishedArticleList = async () => {
       try {
-        const headers = JSON.parse(localStorage.getItem("J-tockAuth-Storage"));
-        const response = await axios.get(
-          "/admin/articles",
-          {},
-          {
-            headers: headers,
-          }
-        );
+        let headers = JSON.parse(localStorage.getItem("J-tockAuth-Storage"));
+        headers = {
+          ...headers,
+          "Content-type": "application/json",
+          Accept: "application/json",
+        };
+        const response = await axios.get("/admin/articles", {
+          headers: headers,
+        });
         setUnpublishedArticleList(response.data.articles);
       } catch (error) {
         console.log(error);
```

Comment:
We should probably display error messages if there would be any.

Reply:
Now there is a general message saying you should pick something and if you get an error this message changes to be the received error instead.